tests/conftest.py

```diff
@@ -1,10 +1,10 @@
 import pytest
-from flask_pymongo import PyMongo
-from api import create_app
+from api import create_app, mongo
 
-@pytest.fixture(scope="module")
+@pytest.fixture(scope="module", autouse=False)
```

Comment:
Why are we explicitly setting autouse to false?

Reply:
So we only want to delete the database after every single testing has been done. autouse by default is set to True, meaning that each testing file, the DB gets wiped, but autouse=False (i might have to do a bit more research) ensures that DB gets wiped after all the testing files have been gone through.